### PR TITLE
Fix/isolated networks

### PR DESCRIPTION
**Pull request by Thiht**
Don't overwrite progress to 0 if an ISOLATED_NETWORK service is deployed in Docktor.

### CODE REVIEW

File: server/docktor/docktor.go [hunk 1]
```diff
@@ -15,77 +15,83 @@ import (
...
-	docktorAddr     string
-	docktorUser     string
-	docktorPassword string
+	address  string
+	username string
+	password string
 }
 
 // NewExternalAPI create a new ExternalAPI from authentication information
-// addr is the address of the Docktor instance
```
Comment (Ezian) [inline server/docktor/docktor.go]: ❓ Why remove the documentation ?

File: server/jobs/deploy.go [hunk 2]
```diff
@@ -13,35 +13,45 @@ import (
...
+		return docktor.GroupDocktor{}, err
 	}
 
-	log.Infof("Services availables : %s", docktorGroup.Containers)
+	return docktorGroup, nil
+}
+
+func isIsolatedNetwork(docktorGroupData docktor.GroupDocktor) bool {
+	for _, container := range docktorGroupData.Containers {
+		if container.ServiceTitle == "ISOLATED_NETWORK" {
```
Comment (Ezian) [inline server/jobs/deploy.go]: 🐳 Detection of isolated network is done by finding a container which is named "ISOLATED_NETWORK" ? That is it ?

File: server/jobs/deploy.go [hunk 3]
```diff
@@ -150,11 +165,14 @@ func ExecuteDeploymentStatusAnalytics() (string, error) {
 		constructFullMatrix(&project, functionalServices)
 
 		// Put all the no deployed services to a progress of 0
+		// If the project is isolated, don't touch anything
+		if !isIsolatedNetwork(docktorGroupData) {
```
Comment (Ezian) [inline server/jobs/deploy.go]: ⚠️ Indentation.

File: server/docktor/docktor.go (hunk 1 above)
Comment (Thiht) [reply server/docktor/docktor.go]: The comment doesn't bring more value than the pair package name/variable name

File: server/jobs/deploy.go (hunk 2 above)
Comment (Thiht) [reply server/jobs/deploy.go]: Yup, Isolated Networks are declared in Docktor by us

**Comment by matcornic** [comment]
A part le commentaire qui ne sert plus à rien, ça me semble bien

**Pull request merged**